prow/config/cache.go

```diff
@@ -167,6 +188,47 @@ func NewInRepoConfigCache(
 		return nil, err
 	}
 
+	// This records all OrgRepos we've seen so far during the lifetime of the
+	// process. The main purpose is to allow reporting of 0 counts for OrgRepos
+	// whose keys have been evicted by the lruCache.
```

Comment:
I guess this could just be all the org repos we've last written a positive value for. Once we set the gauge value to 0 we can ignore the repo until there is activity again.